main/auth/external_login/ldap.inc.php

```diff
@@ -405,3 +411,92 @@ function extldap_add_user_by_array($data, $update_if_exists = true)
 
     return $user_id;
 }
+
+/**
+ * Get one user's single attribute value.
+ * User is identified by filter.
+ * $extldap_config['filter'] is also applied in complement, if defined.
+ *
+ * @param $filter string LDAP entry filter, such as '(uid=10000)'
+ * @param $attribute string name of the LDAP attribute to read the value from
+ * @return string|bool the single matching user entry's single attribute value or false if not found
+ * @throws Exception if more than one entries matched or on internal error
+ */
+function extldap_get_user_attribute_value($filter, $attribute)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $extldap_config is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=4906847)